### PR TITLE
Instrumental graphing

### PR DESCRIPTION
**Pull request by netshade**
Add Instrumental graphing support to Hubot

### CODE REVIEW

**Comment by netshade** [comment]
Lots of changes to get reliable rebuilds, node / buffertools / libxmljs buidls failing on Heroku for some reason. Sorry for commit mess, difficult to debug. 

Wolfram temporarily removed to sidestep libxmljs failures.

**Comment by janxious** [comment]
sad to lose this functionality I didn't know existed

File: scripts/instrumental.coffee [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+#   Instrumental Graphs
+#
+#   Set the environment variable HUBOT_INSTRUMENTAL_GRAPH_EMBED_TOKENS
+#   to a comma separated list of room names and embed tokens
+#
+# Dependencies:
+#   moment.js
+#
+# Configuration:
+#   HUBOT_INSTRUMENTAL_GRAPH_EMBED_TOKENS
```
Comment (janxious) [inline scripts/instrumental.coffee:11]: ~~This is just the value from the Instrumental dashboard, right?~~
I see how this is being used now. Clearly the bits form the flowdock room settings. Derp.

**Comment by janxious** [comment]
something something extract methods :+1:

**Comment by tonydewan** [comment]
This is awesome!

**Pull request merged**